_includes/footer.html

```diff
@@ -83,23 +83,15 @@
 
 <!-- Digital Analytics Program roll-up, see https://analytics.usa.gov for data -->
 <script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&subagency=TTS"></script>
-<!-- Google Analytics -->
 
+<!-- Google tag (gtag.js) -->
+<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.analytics['ga_prod'] }}"></script>
 <script>
-(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-{% if site.environment == 'staging' %}
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
 
-{% else %}
-ga('create', '{{ site.analytics['ga_prod'] }}', 'auto');
-// anonymize user IPs (chops off the last IP triplet)
-ga('set', 'anonymizeIp', true);
```

Comment:
@btylerburton @dlennox24  I think this needs to be kept if it's still supported in the same way as the old google analytics...

>All information accessed through Data.gov must be in compliance with current privacy requirements including OMB guidance
source: https://data.gov/privacy-policy/#data-policy

>The White House uses a third-party analytics provider (currently Google Analytics) to analyze data from cookies. The third-party analytics provider does not receive personally identifiable information through these cookies. We have also limited the provider’s ability to see your full IP address (a process known as “IP masking”). Please review Google Analytics’ privacy policy for additional information.
source: https://www.whitehouse.gov/privacy/

Reply:
GA4 doesn't collect IPs by default, thankfully.

> **Google Analytics 4 properties**
In Google Analytics 4, IP masking is not necessary since IP addresses are not logged or stored.

https://support.google.com/analytics/answer/9019185#IP